internal/pkg/earpointer/analysis.go

```diff
@@ -592,10 +692,12 @@ func (vis *visitor) processIndex(data ssa.Value, base ssa.Value, index ssa.Value
 	var field Field
 	if c, ok := index.(*ssa.Const); ok {
 		field = Field{Name: c.Value.String()}
+		vis.processHeapAccess(data, base, field)
```

Comment:
Same comment as for `IndexAddr`: why not do just the "any" field?

Reply:
See above discussions.